### PR TITLE
docs(bridging): how to bridge using etherscan

### PR DESCRIPTION
**Pull request by emilyJLin95**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [ ] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [ ] You've previewed your changes locally.

## Describe the change

Updated the links that went to blockscout and changed them to the goerli.lineascan.build

## Issue fixed

No current issue exists

## Impacted parts

For content changes:

- [ x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

File: docs/use-linea/bridge-funds/use-etherscan.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <td align="left"><b>sendMessage</b></td>
+        <td align="left">This is the amount of ETH to send</td>
+    </tr>
+    <tr>
+        <td align="left"><b>_to</b></td>
+        <td align="left">This is the recipient on the other layer (usually your Ethereum address)</td>
+    </tr>
+    <tr>
+        <td align="left"><b>_fee</b></td>
+        <td align="left">This is the fee paid to the postman to execute the transfer on the second layer automatically. Users will eventually receive payableAmount. We recommend providing a 0.001 ETH fee to ensure proper execution. Note: the amount is in wei, so 0.001 ETH = 1015 or 1000000000000000</td>
```
Comment (arthur-remy) [inline docs/use-linea/bridge-funds/use-etherscan.md:33]: "0.001 ETH = 1015" -> we should add an exponent, it's 10 power 15

File: docs/use-linea/bridge-funds/use-etherscan.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+## Manually complete a bridge transaction
+
+:::note This step is not mandatory unless you didn’t send enough fees when initiating your bridge transaction :::
+
+1. Browse your `Transaction Details` page of the transaction you sent on the source layer
+1. Go to the `Logs` tab
+1. Find the log for the `MessageSent` event and keep it open for a later step ![message sent](../../assets/etherscan-bridge/logs.png)
+1. Go to the [Message Service contract](../../developers/bridge-architecture/message-service.mdx#contracts) on the other Layer ([0xC499a572640B64eA1C8c194c43Bc3E19940719dC](https://goerli.lineascan.build/address/0xC499a572640B64eA1C8c194c43Bc3E19940719dC) on L2)
+1. Navigate to the `Write Proxy` tab, connect your wallet, and call the `claimMessage` function using the parameters found in the logs MessageSent checked earlier. Note: `bytes` argument need to start with `0x0` ![write claimMessage](../../assets/etherscan-bridge/claimmessage.png)
+1. Confirm the transaction on your wallet
```
Comment (arthur-remy) [inline docs/use-linea/bridge-funds/use-etherscan.md:107]: All the paragraph is included in the note above. I don't know if it's intended

Comment (emilyJLin95) [reply docs/use-linea/bridge-funds/use-etherscan.md:107]: ? I don't understand this comment

Comment (emilyJLin95) [reply docs/use-linea/bridge-funds/use-etherscan.md]: whoops that was weird autoformatting

**Review by mapachurro — APPROVED** [review]

**Comment by mapachurro** [comment]
Hoping this brings things back into alignment.

**Pull request merged**